### ISSUE TITLE
"Load more posts" causes stutter on large topics

### ISSUE THREAD

**Issue by julianlam** [NodeBB/NodeBB#4477]
If browsing near the end of a topic, NodeBB will make a request to the server to load more posts.

Having a high post count in a topic causes `loadMorePosts` to take longer to complete (for whatever reason)

Measuring code:
```
require(['forum/infinitescroll'], function(is) {
  console.time('test');
  is.loadMore('topics.loadMore', {
    tid: 16534,
    after: 30,
    direction: undefined
  }, function() {
    console.timeEnd('test');
  });
});
```

tid `16534` with 2600+ posts: 234ms
tid `19481` with 77 posts: 65ms

**Comment by julianlam** [comment]
On an unrelated note, scrolling up and down _near_ the bottom causes `loadMorePosts` to be called repeatedly, so scrolling at the bottom of a very large topic is very stuttery...

**Comment by pikshub** [comment]
categories have problems. Scroll does not work.More is not shown post.

nodebb.min.js?c1fe9479-50bb-459e-bab9-acc636fa7915:16 Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined

**Comment by julianlam** [comment]
That is a separate issue, but please try with above commit.

**Comment by pikshub** [comment]
Sorry.

Fix the problem. Thank you so much.
